internals/secrethub/run.go

```diff
@@ -81,7 +80,8 @@ func (cmd *RunCommand) Register(r command.Registerer) {
 	clause.Alias("exec")
 	clause.Arg("command", "The command to execute").Required().StringsVar(&cmd.command)
 	clause.Flag("no-masking", "Disable masking of secrets on stdout and stderr").BoolVar(&cmd.noMasking)
-	clause.Flag("masking-timeout", "The maximum time output is buffered. Warning: lowering this value increases the chance of secrets not being masked.").Default("1s").DurationVar(&cmd.maskingTimeout)
+	clause.Flag("no-output-buffering", "Disable output buffering. This increases output responsiveness, but decreases the probability that secrets get masked.").BoolVar(&cmd.maskerOptions.DisableBuffer)
+	clause.Flag("masking-buffer-period", "The time period for which output is buffered. A higher value increases the probability that secrets get masked but decreases output responsiveness.").Default("50ms").DurationVar(&cmd.maskerOptions.BufferDelay)
```

Comment:
What's the difference between `--no-output-buffering` and `--masking-buffer-period 0`?

Reply:
`--no-output-buffering` removes the buffer -> input blocks until flushed til output.
`--masking-buffer-period 0` directly flushes output, but there still is a buffer.

Reply:
Shall we use `input blocks` or something similar in the `--no-output-buffering` help text to clarify this difference to the user?

Reply:
Do you have a suggestion on how to phrase this that it makes things clearer? Just saying that the "input blocks" is confusing when you're reading the flag description ("what input? What does it have to do with _output_ buffering?"). 

I have the feeling that it quickly makes things confusing and that people who care about this, can derive this information from the first part ("disable buffering").

Reply:
Yes, I think you're right. It's very hard to get this across in simple words.

Let's leave it out indeed. Wouldn't want to confuse the masses to explain it for the few people that actually need this behavior.